src/FuseFlywheelBorrowerCore.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.10;
+
+import "flywheel/FlywheelCore.sol";
+
+contract FuseFlywheelCore is FlywheelCore {
```

Comment:
Can we rename this to FuseFlywheelBorrowCore ?

Reply:
sure